bundles/action/org.openhab.action.pushbullet/README.md

```diff
@@ -2,6 +2,34 @@
 
 The Pushbullet action allows you to notify iOS, Android & Windows 10 Phone & Desktop devices of a message using the Pushbullet API web service.
 
+## Configuration
+
+You may define default values for parameters to action calls in the file `services/pushbullet.cfg`.
+The configuration parameter `accesstoken` is required and can be obtained as described below.
+
+| Property            | Default | Required                        | method to set         | Description                                                                                                   |
+|---------------------|---------|:-------------------------------:|-----------------------|---------------------------------------------------------------------------------------------------------------|
+| accesstoken         | 1234abc | Yes                             | -                     | Pushbullet [API token](https://www.pushbullet.com/#settings) to send to devices       |

```

Comment:
when you install the action, the deployed `conf/services/pushbullet.cfg` includes by default the following parameter set:

```
accesstoken=1234abc
```

Maybe this should be commented out and used as an example in the distributed `pushbullet.cfg`
But for now, since this parameter is active, I considered it as a default.

This is wrong, so I removed the default value from the table (and kept the requirement) :)